store/tikv/tikvrpc/tikvrpc.go

```diff
@@ -189,177 +189,212 @@ func NewReplicaReadRequest(typ CmdType, pointer interface{}, replicaReadType kv.
 
 // Get returns GetRequest in request.
 func (req *Request) Get() *kvrpcpb.GetRequest {
-	return req.req.(*kvrpcpb.GetRequest)
+	r, _ := req.req.(*kvrpcpb.GetRequest)
```

Comment:
The old code could panic, but the new code we don't know what is wrong.

Reply:
It will be a segfault due to dereferencing a nil pointer. That will still cause an error, although not as close to the cause. I guess the alternative is to make the req field public and do the type conversation at the call site. Do you prefer that?

Reply:
I've made the change in the last commit